index.js

```diff
@@ -223,14 +223,25 @@ module.exports = function Devtools () {
       return services
     }
 
-    controller.on('modulesLoaded', function () {
+    function start () {
       if (window.__CEREBRAL_DEVTOOLS_GLOBAL_HOOK__) {
         window.__CEREBRAL_DEVTOOLS_GLOBAL_HOOK__.signals = controller.getSignals()
       }
 
       var event = new CustomEvent('cerebral.dev.cerebralPing')
       window.dispatchEvent(event)
+
+      console.assert(controller.listeners('modulesLoaded')[0] === start, 'Cerebral devtools: Please do not place any listeners to `modulesLoaded` event before devtools\'s one.')
```

Comment:
Looks good! Though, not sure why we need this one? Since you put them in the correct order below, this will never trigger?